rsocket/statemachine/RSocketStateMachine.cpp

```diff
@@ -100,8 +100,8 @@ bool RSocketStateMachine::resumeServer(
   if (!isDisconnected()) {
```

Comment:
Should this be the first thing to do, namely should we immediately check if it is not disconnected and then perform the other calculations?

Reply:
The calculations are for logging.  We do them irrespective of the outcome of `isDisconnected()`